generators/ionic/resources/base/cypress.config.ts

```diff
@@ -16,7 +16,7 @@ export const defaultConfig = {
       return (await import('./cypress/plugins/index')).default(on, config);
     },
     baseUrl: 'http://localhost:8100/',
-    specPattern: '**/*.e2e-spec.ts',
+    specPattern: '**/*.cy.ts',
```

Comment:
I think this can be dropped should be the default.

Reply:
Are you saying the `specPattern` line is not necessary?

Reply:
Yes.